src/test/script_tests.cpp

```diff
@@ -1610,5 +1640,50 @@ BOOST_AUTO_TEST_CASE(bitcoinconsensus_verify_script_invalid_flags)
     BOOST_CHECK_EQUAL(err, bitcoinconsensus_ERR_INVALID_FLAGS);
 }
 
+BOOST_AUTO_TEST_CASE(script_taproot)
+{
+    UniValue tests = read_json(std::string(std::begin(json_tests::script_taproot_tests), std::end(json_tests::script_taproot_tests)));
+
+    static constexpr unsigned int VERIFY_FLAGS =
+        SCRIPT_VERIFY_P2SH |
+        SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY |
+        SCRIPT_VERIFY_CHECKSEQUENCEVERIFY |
+        SCRIPT_VERIFY_WITNESS |
+        SCRIPT_VERIFY_NULLFAIL |
+        SCRIPT_VERIFY_TAPROOT;
```

Comment:
Why no `MINIMALIF`?

Reply:
... because it's not used for consensus, along with CLEANSTACK et al. Right.

Reply:
Added a comment in script/interpreter.h.